packages/token-sdk/src/repository.ts

```diff
@@ -173,7 +165,7 @@ export class TokenRepository {
    * @returns Token metadata and tags for all mints with the given tag. Excluded mints are not
    * returned.
```

Comment:
I think the excluded feature has been removed.
Wouldn't a filter based on ``exists`` be necessary to implement "If a mint is not in the repository, it is not returned"?